openapi/task_execution_service.openapi.yaml

```diff
@@ -219,6 +236,19 @@ components:
         - FULL
 
   schemas:
+    tesCallbackStatus:
```

Comment:
How will the the callback know which TES server is making the request? Is the callback expected to look at the Origin header (but I don't think that's guaranteed to be set)?